bin/i18n/test/translation_monitor/test_translation_validator.rb

```diff
@@ -0,0 +1,46 @@
+require_relative '../../translation_monitor/translation_validator'
+require_relative '../test_helper'
+
+class TranslationValidatorTest < Minitest::Test
+  include TranslationValidator
+
+  def test_validate_markdown_link
+    # valid cases
+    assert_nil validate_markdown_link('[text](url)')
+    assert_nil validate_markdown_link('[.](http://go.somewhere)')
+
+    # invalid cases
+    refute_nil validate_markdown_link('[text] (url)')
+    refute_nil validate_markdown_link("[text]\t(url)")
+  end
+
+  def test_validate_redacted_blocks
+    # valid cases
+    assert_nil validate_redacted_blocks('[A][0]')
+    assert_nil validate_redacted_blocks('[A][0] [B][1]')
```

Comment:
Do we ever have no space between two redacted blocks? If so is that accounted for?

Reply:
good idea, I'll add that test